daemon/main.go

```diff
@@ -245,6 +246,8 @@ func init() {
 		"enable-tracing", false, "Enable tracing while determining policy (debugging)")
 	flags.StringVar(&v4Prefix,
 		"ipv4-range", "auto", "Per-node IPv4 endpoint prefix")
+	flags.StringVar(&v6Prefix,
+		"ipv6-range", "auto", "Per-node IPv6 endpoint prefix, must be /96")
```

Comment:
What was the outcome of running it with a link-local ipv6 address does cilium handle it well?

also:
`"Per-node IPv6 endpoint prefix, must be /%d", IPv6NodePrefixLen`?